packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -817,8 +812,6 @@ class _SelectionHandleOverlayState extends State<_SelectionHandleOverlay> with S
     final Offset handleAnchor = widget.selectionControls.getHandleAnchor(
       widget.type,
       widget.preferredLineHeight,
-      widget.glyphHeight,
-      widget.glyphHeight,
     );
     final Size handleSize = widget.selectionControls.getHandleSize(
```

Comment:
I think this should have used glyphHeight instead of lineHeight. The size is used to pad gesture rect to make it easier to tap. This refactor also fixes it because the `preferredLineHeight` know actually contains the glyphHeight